src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -107,11 +107,10 @@ namespace Akka.Actor
         protected void SetTerminated() { }
         public static Akka.Actor.NameAndUid SplitNameAndUid(string name) { }
         public virtual void Start() { }
+        protected void Stash(Akka.Dispatch.SysMsg.SystemMessage msg) { }
```

Comment:
Internal stash for `SystemMessage`s that can't be processed during different types of suspensions